src/Transform.jl

```diff
@@ -890,7 +890,7 @@ function enforce_bounds!{NumType <: Number}(
         if !(constraint.lower_bound <=
              mp.vp[s][ids.(param)[ind]] <=
              constraint.upper_bound)
-          println("Warning: $param[$s][$ind] was out of bounds.")
+          info("Warning: $param[$s][$ind] was out of bounds.")
```

Comment:
I was just removing all the println()s. I'll let you guys decide if these info/warning/print messages are required.